package.json

```diff
@@ -2,7 +2,7 @@
   "name": "nebenan-redux-tools",
   "description": "A set of redux helpers",
   "author": "Good Hood GmbH <tech@nebenan.de>",
-  "license": "SEE LICENSE IN LICENSE FILE",
+  "license": "MIT",
```

Comment:
I think we need to talk to someone in legal about this.

Reply:
i second this, we should clarify this for all public repos

Reply:
talked with lars about it, he's ok with it